JSQMessagesViewController/Model/JSQLocationMediaItem.m

```diff
@@ -160,6 +162,15 @@ - (NSUInteger)mediaHash
     return self.hash;
 }
 
+- (NSDictionary *)copyableMediaItem {
+    // NSString *locationAsAppleMapsString = [NSString stringWithFormat:@"http://maps.apple.com/?ll=%f,%f", self.coordinate.latitude, self.coordinate.longitude];
```

Comment:
using an Apple Maps URL would be preferable as a default value. Were there issues with this? 
I'm assuming that taping this link will open Apple maps?

It would be nice to allow consumers to choose. How about this:

1. Create a new `JSQLocationMediaItemURLBuilder` class
2. You can initialize the builder with an enum (`JSQLocationURLAppleMaps` or `JSQLocationURLGoogleMaps`)
3. Then, ask the builder for the URL: `NSURL *url = [builder urlForForCoordinate:coord];`